internal/providers/bedrock_mantle.go

```
@@ -0,0 +1,626 @@
+	return payload.User
+}
+
+// ValidateAPIKey accepts llm-proxy keys registered for the Bedrock family.
+// OpenAI clients send “Authorization: Bearer“; Anthropic Messages clients
+// (langchain_anthropic / the Anthropic SDK) send “x-api-key“. Both are
+// accepted and stripped before SigV4 signing — Mantle upstream auth is AWS
+// credentials only.
+func (b *BedrockMantleProxy) ValidateAPIKey(req *http.Request, keyStore APIKeyStore) error {
+	key := mantleProxyKeyFromRequest(req)
+	if key == "" {
+		return fmt.Errorf("Bearer proxy API key is required")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)